diesel/src/sqlite/types/date_and_time/time.rs

```diff
@@ -14,23 +14,30 @@ use crate::serialize::{self, IsNull, Output, ToSql};
 use crate::sql_types::{Date, Time, Timestamp, TimestamptzSqlite};
 use crate::sqlite::Sqlite;
 
+// Warning to future editors:
+// Changes in the following formats need to be kept in sync
+// with the formats of the "chrono" module.
+// We need a distinction between whole second and subsecond
+// since there is no format option to forgo the dot.
+// We always print as many subsecond as his given to us,
+// this means the subsecond part can be between 1 and 9 digits.
```

Comment:
```suggestion
/// Warning to future editors:
/// Changes in the following formats need to be kept in sync
/// with the formats of the ["chrono"](super::chrono) module.
/// We need a distinction between whole second and subsecond
/// since there is no format option to forgo the dot.
/// We always print as many subsecond as his given to us,
/// this means the subsecond part can be between 1 and 9 digits.
```